src/components/shell-panel/shell-panel.scss

```diff
@@ -52,15 +67,15 @@
   --calcite-shell-panel-min-width-internal: var(--calcite-shell-panel-min-width, 340px);
 }
 
-:host([detached-height-scale="s"]) .content--detached {
+:host([detached-height-scale="s"]) .content--float {
```

Comment:
Should we update this detached prop to say float as well? deprecate and add new?

Reply:
Or could we just deprecate this prop and use a CSS var instead?

Reply:
I think we could look at changing how this behaves in the future, there’s a need for a “full” and “auto” height. “Float” sounds a bit weird in this context but if that’s the value name we should update to match.

Reply:
@jcfranco should we deprecate `detatchedHeightScale` in favor of a new prop?

Reply:
Good point. I believe we should once we have the new prop in place.

Reply:
I wonder if we need this at all with the available css variables, especially if it only affects the one mode. 

Detached shell panel in vertical layout could be height: auto to fit content, and could set a max height  of 100% (minus margin). Not sure how common having a defined height that is less than 100% but more than the height of the content is.

cc @asangma for general Shell FYI / use cases

Reply:
I added it but if we want to remove we can.

Reply:
@macandcheese do you want me to remove the `heightScale` prop and just make it so a user can control the height via a CSS variable?